### PR TITLE
Iw v2

### PR DESCRIPTION
**Pull request by stafaniasaju**
### Description

The updates for PowerVS Infrastructure flavour 'import-workspace'. 

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by surajsbharadwaj — CHANGES_REQUESTED** [review]

**Review by surajsbharadwaj — CHANGES_REQUESTED** [review]
Please correct them

File: reference-architectures/import-workspace/deploy-arch-ibm-pvs-inf-import-workspace.md
Comment (surajsbharadwaj) [inline reference-architectures/import-workspace/deploy-arch-ibm-pvs-inf-import-workspace.md:1]: correct this file with headings

File: ibm_catalog.json [hunk 1]
```diff
@@ -947,42 +947,27 @@
...
-                                "profile_version": "1.3.0"
-                            }
-                        ]
-                    },
+                    "compliance": {},
                     "configuration": [
                         {
                             "key": "access_host",
                             "required": true,
+                            "default_value": "{\"vsi_name\":\"\",\"floating_ip\":\"\"}",
```
Comment (surajsbharadwaj) [inline ibm_catalog.json:960]: remove line default_value

File: ibm_catalog.json [hunk 2]
```diff
@@ -947,42 +947,27 @@
...
-                                "grouping": "deployment",
-                                "original_grouping": "deployment",
-                                "type": "json_editor"
-                            }
-                        },
-                        {
-                            "key": "workload_host",
+                            "key": "proxy_server_ip_port",
                             "required": true,
+                            "default_value": "{\"vsi_ip\":\"\",\"port\":\"\"}",
```
Comment (surajsbharadwaj) [inline ibm_catalog.json:970]: remove line default_value

File: ibm_catalog.json [hunk 3]
```diff
@@ -1246,7 +1238,19 @@
...
                             "service_name": "power-iaas"
                         }
-                    ]
+                    ],
+                    "architecture": {
+                        "diagrams": [
+                            {
+                                "diagram": {
+                                    "caption": "Power Virtual Server with VPC landing zone 'import-workspace' variation",
+                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/iw_v2/reference-architectures/import-workspace/deploy-arch-ibm-pvs-inf-import-workspace.svg",
```
Comment (surajsbharadwaj) [inline ibm_catalog.json:1247]: change this to main branch

File: ibm_catalog.json [hunk 4]
```diff
@@ -1246,7 +1238,19 @@
...
+                                    "type": "image/svg+xml"
+                                },
+                                "description": "The Power Virtual Server with VPC landing zone as variation 'Create a new architecture' - 'import-workspace' imports existing VPC services and interconneted Power Virtual Server workspace.\n \nRequired and optional management components are configured."
```
Comment (surajsbharadwaj) [inline ibm_catalog.json:1250]: Change the description

File: reference-architectures/full-stack/deploy-arch-ibm-pvs-inf-full-stack.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ A proxy service for public internet access from the PowerVS workspace is configu
 ## Design requirements
 {: #full-design-requirements}
 
-![Design requirements for 'Power Virtual Server with VPC landing zone' - variation 'PowerVS workspace'.](heat-map-deploy-arch-ibm-pvs-inf-full-stack.svg "Design requirements"){: caption="Figure 2. Scope of the solution requirements" caption-side="bottom"}
+![Design requirements for 'Power Virtual Server with VPC landing zone' - variation 'PowerVS Import workspace'.](heat-map-deploy-arch-ibm-pvs-inf-full-stack.svg "Design requirements"){: caption="Figure 2. Scope of the solution requirements" caption-side="bottom"}
```
Comment (surajsbharadwaj) [inline reference-architectures/full-stack/deploy-arch-ibm-pvs-inf-full-stack.md:66]: Revert it back

File: solutions/import-workspace/outputs.tf
Comment (surajsbharadwaj) [inline solutions/import-workspace/outputs.tf:1]: remove created in all descriptions

File: solutions/import-workspace/provider.tf [hunk 6]
```diff
@@ -1,48 +1,53 @@
...
     "dal10"    = "us-south"
     "dal12"    = "us-south"
     "us-east"  = "us-east"
     "wdc06"    = "us-east"
+    "wdc07"    = "us-east"
   }
 }
 
 provider "ibm" {
   region           = lookup(local.ibm_powervs_zone_cloud_region_map, var.powervs_zone, null)
```
Comment (surajsbharadwaj) [inline solutions/import-workspace/provider.tf:50]: zone is missing

File: solutions/import-workspace/variables.tf [hunk 7]
```diff
@@ -1,51 +1,29 @@
...
-  default = {
-    "vsi_name" : "",
-    "floating_ip" : ""
-  }
 }
 
-variable "proxy_host" {
-  description = "Name of the existing VSI on which proxy server is configured and proxy server port."
+variable "proxy_server_ip_port" {
+  description = "Proxy Server IP and port. This will be required to configure internet access for PowerVS instances."
```
Comment (surajsbharadwaj) [inline solutions/import-workspace/variables.tf:14]: Existing proxy server

File: solutions/import-workspace/variables.tf [hunk 8]
```diff
@@ -69,15 +47,53 @@ variable "powervs_sshkey_name" {
 }
 
 variable "powervs_management_network_name" {
-  description = "Name of management network in existing PowerVS workspace."
+  description = "Name of the subnet used for management network in existing PowerVS workspace."
```
Comment (surajsbharadwaj) [inline solutions/import-workspace/variables.tf:50]: Name of the existing subnet..

File: solutions/import-workspace/variables.tf [hunk 9]
```diff
@@ -69,15 +47,53 @@ variable "powervs_sshkey_name" {
...
   type        = string
 }
 
 variable "powervs_backup_network_name" {
-  description = "Name of backup network in existing PowerVS workspace."
+  description = "Name of the subnet used for backup network in existing PowerVS workspace."
```
Comment (surajsbharadwaj) [inline solutions/import-workspace/variables.tf:55]: Name of the existing subnet..

**Review by surajsbharadwaj — CHANGES_REQUESTED** [review]
Some more changes

File: solutions/import-workspace/provider.tf (hunk 6 above)
Comment (stafaniasaju) [reply solutions/import-workspace/provider.tf:50]: Zone is not required for VPC resources we are using

File: ibm_catalog.json (hunk 1 above)
Comment (stafaniasaju) [reply ibm_catalog.json:960]: Discussed and decided not to change.

File: ibm_catalog.json (hunk 4 above)
Comment (stafaniasaju) [reply ibm_catalog.json:1250]: done

File: solutions/import-workspace/outputs.tf
Comment (stafaniasaju) [reply solutions/import-workspace/outputs.tf:1]: done

**Review by surajsbharadwaj — APPROVED** [review]

**Pull request merged**